{
  "id": "CVE-2023-28953",
  "sourceIdentifier": "psirt@us.ibm.com",
  "published": "2023-07-10T16:15:50.257",
  "lastModified": "2023-07-10T16:27:17.833",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "IBM Cognos Analytics on Cloud Pak for Data 4.0 could allow an attacker to make system calls that might compromise the security of the containers due to misconfigured security context.  IBM X-Force ID:  251465."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "psirt@us.ibm.com",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:L/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "HIGH",
          "privilegesRequired": "LOW",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "NONE",
          "integrityImpact": "LOW",
          "availabilityImpact": "NONE",
          "baseScore": 3.1,
          "baseSeverity": "LOW"
        },
        "exploitabilityScore": 1.6,
        "impactScore": 1.4
      }
    ]
  },
  "references": [
    {
      "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/251465",
      "source": "psirt@us.ibm.com"
    },
    {
      "url": "https://www.ibm.com/support/pages/node/7006413",
      "source": "psirt@us.ibm.com"
    }
  ]
}